app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -6,8 +6,6 @@
   logo_text = t("components.layout_super_navigation_header.logo_text")
 
   hide_logo_text ||= false
-  blue_background ||= false
```

Comment:
Not sure if we still need this option - it looks like it's used in a few places:
```
  top_toggle_button_classes << "gem-c-layout-super-navigation-header__navigation-top-toggle-button--blue-background" if blue_background
```
If it’s okay to remove, we might want to do that in a separate commit.

Reply:
Ah, I see - there's a separate commit for removing the blue background styles. Perhaps this could be included with those changes instead.

Reply:
Ah sorry, I included this in the wrong commit, I've fixed this now and moved it to the commit which removes the blue background styles